lib/schemas/interlock-constructor-input.js

```diff
@@ -0,0 +1,41 @@
+import * as path from "path";
```

Comment:
If schemas are to be used for testing or if there will be many of them it may make sense to have a directory structure that mirrors `/lib` in the same way that `/test` does. For that reason it may be better to move this to the project root even though it does add a tiny bit of clutter. This is especially true if you use generative testing and end up with an input and an output schema for each publicly exposed function.